typescript/src/lib/services/shared_path.ts

```diff
@@ -0,0 +1,35 @@
+import * as path from "path"
+
+/*
+Represents and absolute path shared between two containers, it could be a file, a directory or a symlink
```

Comment:
Same thing about a docs link (reason being, then you don't have to copy-paste this comment across all languages)